### PR TITLE
Disable views for course in React if they are disabled on back end

### PR DESCRIPTION
**Pull request by justin0022**
Fixes https://github.com/tl-its-umich-edu/my-learning-analytics/issues/469.

I included updates from master branch into this PR.

### CODE REVIEW

File: assets/src/routes/routes.js [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-]
+const routes = (courseId, activeViews) => {
+  const allViews = [
+    {
+      path: `/${courseId}/grades`,
+      title: 'Grade Distribution',
+      icon: Grade,
+      description: 'See where your grade sits within the course grade distribution.',
+      image: '/static/images/grade_distribution_icon.png',
+      viewCode: 'gd'
```
Comment (justin0022) [inline assets/src/routes/routes.js]: This is slightly _hacky_, based on the JSON sent back from the `/info` endpoint. Could be improved/modified in the future if we change the APIs.

**Comment by justin0022** [comment]
Just a note, both the `IndexPage` and the side drawer should show only the views that are activated for the course. `IndexPage` url is `/test/courses/{courseid}`.

**Comment by pushyamig** [comment]
Previous commit history should not be added for this PR, I feel there is something wrong the way this branch is created

**Comment by justin0022** [comment]
The previous commits are from the `master` branch. Should I update the `react` branch from `master` first, then make this PR?

**Comment by pushyamig** [comment]
I think Andrew already pulled those commits into upstream react branch. You might need to pull the latest changes from the upstream react branch to your local forked branch and them make a new branch from there to not see the commit history.

**Comment by justin0022** [comment]
Okay sure I'll try again then

**Pull request closed without merging**